librz/main/rz-find.c

```diff
@@ -140,6 +142,15 @@ static int hit(RzSearchKeyword *kw, void *user, ut64 addr) {
 			}
 		}
 	}
+	if (ro->exec_command) {
+		char *command = rz_str_newf("%s", ro->exec_command);
+		int status = rz_sys_system(command);
+		if (status == -1) {
+			RZ_LOG_ERROR("Failed to execute command: %s", command);
```

Comment:
shouldn't this be executed with the found file?

Reply:
i think that is why you had the issue before where the double `grep` didn't work.

Reply:
It's is but ig we are filtering the output w `~?` to return the number of times the string has occurred because without `~?`, I do get all instances of the string I'm searching.

Reply:
yes, but the command should be executed with the file name, that is why you got `0` hit with the double grep.

Reply:
```sh
[XX] C:\projects\rizin\test\db\tools\rz_find rz-find -E with echo as a command
ANSICON=1 RZ_NOPLUGINS=1 C:\projects\rizin\rizin-vs2022_64-v0.8.0\bin\rizin.exe -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc '!rz-find -s "README" -E "echo" bins/arm~?' =
-- stdout
--- expected
+++ actual
@@ -1,1 +1,1 @@
-1
+2
-- stderr
```
I've for some reason, this returning me `2` on windows instead of `1`.

Reply:
https://github.com/rizinorg/rizin/pull/4200/commits/4eb0d284adde63abcd8e0c39522d6585df5610f4
This failed to return `1` because you removed the file name.

Reply:
The code should be
```c
char *command = rz_str_newf("%s ; grep -c . %s", ro->exec_command, ro->curfile);
```

Reply:
Then this should work
`CMDS=!rz-find -s "README" -E "echo" bins/arm~README~?`

Reply:
Okayyyy got it. Thanks